src/main/webapp/js/grapheditor/Format.js

```diff
@@ -4574,8 +4574,7 @@ StyleFormatPanel.prototype.init = function()
 	var editor = ui.editor;
 	var graph = editor.graph;
 	var ss = this.format.getSelectionState();
-	
-	if (!ss.containsLabel && ss.cells.length > 0)
+	if (!ss.containsLabel && ss.cells.length > 0 && !ss.style.style)
```

Comment:
I guess you should rename this selectionState to avoid the Godwin point

Reply:
Indeed that really well.
But since we work directly on the Fork, we try to make as few changes as possible to maintain the security fixes without too many problems. :/

Reply:
it is named like that in the upstream
https://github.com/jgraph/drawio/blob/dev/src/main/webapp/js/grapheditor/Shapes.js#L5128
we should, as much as possible, keep the code nearer that the upstream

or upstream merge will be painful